src/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export module InteractiveConstants {
```

Comment:
We don't really use modules to declare to `wrap things up` anywhere in TS, namespaces maybe, but not modules.